drive/src/drive/cache.rs

```diff
@@ -0,0 +1,28 @@
+use crate::contract::Contract;
+use crate::drive::flags::StorageFlags;
+use crate::fee::FeeResult;
+use costs::OperationCost;
+use moka::sync::Cache;
+use std::sync::Arc;
+
+/// Drive cache struct
+pub struct DriveCache {
+    /// Cached contracts
+    pub cached_contracts: Cache<[u8; 32], Arc<ContractFetchInfo>>,
+    /// Genesis time in ms
+    pub genesis_time_ms: Option<u64>,
+}
+
+/// Contract and fetch information
+pub struct ContractFetchInfo {
```

Comment:
According to the module hierarchal, this structure should be defined in the contract module where you place fetch functions.

Reply:
you are correct... will fix it.